addons/service.xbmc.versioncheck/resources/lib/version_check/service.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
This change has to be done upstream https://github.com/XBMC-Addons/service.xbmc.versioncheck

Reply:
Done and merged